src/rpc/server.cpp

```diff
@@ -258,6 +258,27 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     return "Bitcoin server stopping";
 }
 
+UniValue uptime(const JSONRPCRequest& jsonRequest)
+{
+    if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
+        throw std::runtime_error(
+                "uptime\n"
+                        "\nDisplay the uptime of the Bitcoin server.");
+
+    int64_t uptime = GetTime() - nStartupTime;
+
+    if (uptime < 0)
+    {
+        throw std::runtime_error(
+                "stop\n"
+                        "\nUptime value for the Bitcoin server is a negative value.");
```

Comment:
Yup, I do not have a problem with this. Just return negative value and do not crash.